godel_process_path_generation/include/godel_process_path_generation/mesh_importer.h

```diff
@@ -75,7 +75,7 @@ class MeshImporter
 
   bool applyConcaveHull(const Cloud& plane_cloud,pcl::ModelCoefficients &plane_coeffs,geometry_msgs::PolygonStamped& polygon);
 
-  void computeLocalPlaneFrame(const Eigen::Hyperplane<double, 3> &plane, const Eigen::Vector3d &centroid);
+  void computeLocalPlaneFrame(const Eigen::Hyperplane<double, 3> &plane, const Eigen::Vector4d &centroid, const Cloud& cloud);
```

Comment:
Never mind, that's just how **pcl::computeCovarianceMatrixNormalized** takes that argument.